src/types/InputValidatorBlock.ts

```diff
@@ -1,9 +1,14 @@
+// src/types/InputValidatorBlock.ts
+
 import { Block } from "react-chatbotify";
 import { ValidationResult } from "./ValidationResult";
 
 /**
  * Extends the Block from React ChatBotify to support inputValidator attribute.
  */
-export type InputValidatorBlock = Block & {
-    validateInput: (userInput?: string) => ValidationResult;
+
+export type InputValidatorBlock = Omit<Block, "file"> & {
+  file?: (params: { files?: FileList }) => void | Promise<void>;
```

Comment:
The `file` attribute is already present in the original `Block` type, not needed here.